src/app/components/score/score.component.ts

```diff
@@ -0,0 +1,18 @@
+import { MatchScore } from './../../models/match-score';
+import { Component, Input, Output } from '@angular/core';
+
+@Component({
+  selector: 'app-score',
+  templateUrl: './score.component.html',
+  styleUrls: ['./score.component.scss']
+})
+export class ScoreComponent {
+
+  @Input()
+  score: MatchScore;
+
+  hover() {
```

Comment:
method seems to be redundant. good practise in order to keep clean code is to not merging empty methods